src/components/scrollable-list/scrollable-list.component.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import tagComponent from '../../utils/helpers/tags';
+import ScrollableListItem from './scrollable-list-item.component';
+import ScrollableListContext from './scrollable-list.context';
+import ScrollableListContainer from './scrollable-list.style';
+import propTypes from './scrollable-list.proptypes';
+
+
+class ScrollableList extends Component {
+  static propTypes = propTypes
+
+  state = {
+    selectedItem: 0
+  }
+
+  scrollBox = React.createRef();
+
+  componentDidMount() {
+    if (this.props.keyNavigation) {
+      document.addEventListener('keydown', this.handleKeyDown);
+    }
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown');
+  }
+
+  updateScroll = (item) => {
+    const { current: list } = this.scrollBox,
+        { offsetHeight: listHeight, children } = list,
+        { offsetHeight: itemHeight, offsetTop: itemTop } = children[item];
+
+    if (itemTop > listHeight) {
+      list.scrollTop = (itemHeight * (item + 1) + 5) - listHeight;
+    } else {
+      list.scrollTop = 0;
+    }
+  }
+
+  handleScroll = ({ target: { scrollTop, scrollHeight } }) => {
+    if (!this.props.onLazyLoad) return;
+
+    if ((scrollHeight - scrollTop) < 200) this.props.onLazyLoad();
+  }
+
+  handleKeyDown = (e) => {
+    e.preventDefault();
+
+    const { selectedItem } = this.state,
+        end = this.props.children.length - 1,
+        atEnd = selectedItem === end,
+        atStart = selectedItem === 0;
+
+    let newPos = selectedItem;
+
+    switch (e.key) {
```

Comment:
I believe we have avoided `.key` at the moment due to IE11 and Edge inconsistencies, see https://caniuse.com/#feat=keyboardevent-key

I'd recommend using our helpers, as we can upgrade their usage as our support updates https://github.com/Sage/carbon/blob/master/src/utils/helpers/events/events.js